common/app/common/Chronos.scala

```diff
@@ -0,0 +1,40 @@
+package common
+
+import org.joda.time.DateTime
+
+import java.text.SimpleDateFormat
+import java.time.ZoneId
+import java.util.TimeZone
+
+object Chronos {
+
+  // Introduced in August 2021 by Pascal to help and support the migration from joda.time to java.time
```

Comment:
If you move the comment directly above the object definition line (rather than a bit below as here) it will come up over hover in editors.

Reply:
Nice one, thanks Nic!